features/step_definitions/gegeven-stepdefs-autorisatie.js

```diff
@@ -20,10 +20,28 @@ Given(/^de geauthenticeerde consumer heeft de volgende 'claim' gegevens$/, funct
         : dataTable.hashes()[0].gemeenteCode;
 });
 
```

Comment:
Melvin, horen we wijzigingen aan step_definitions niet door te voeren in brp-shared en niet in andere repos

Reply:
Klopt, dat hadden we afgesproken. Ik heb deze wijzigingen ook doorgevoerd in brp-shared. Zie: https://github.com/BRP-API/brp-shared/commit/aa1023c10bd12ec77b95bc0d02951a355283bb7b